### ISSUE TITLE
Possible bug related to star expression

### ISSUE THREAD

**Issue by solzard** [psf/black#1497]
Black cannot parse properly the code below, though it runs well.
[Here ](https://black.now.sh/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AEcALxdAD2IimZxl1N_WlbvK5V9KPuirYP5pVyMXDRjI4F3FhWORikO2gz3exV6fe588smwOAzzjEqpZZoS-K28pSGI27ZyxNfRzCP8vOlYzwn7ZG-qxySgChoD9PESZhzA8ywgcRxgDZOROFT-A8hmjHV0cM59qgsHwxpMw74qpZxeCtGg0ivWpOvQbrr1cI-uF7j--7sIQa3fT3WgLihC_14MukUJcIbDUL5h6S_G6e7A6wJ8C5Bgb7xHVpfbOfcKAM_dmWhaNJdPAAHYAZ0CAAA4jvZMscRn-wIAAAAABFla) is the reproduction on Black Playground.
```Python
def func(a):
    x, *y = a[0]
    return -x, *y  # Cannot parse: 3:15:     return -x, *y


def main():
    a = [[1, 2, 3], [4, 5, 6]]
    print(func(a))  # >> (-1, 2, 3)


if __name__ == "__main__":
    main()
```

**Comment by ichard26** [comment]
Fixed since master:
```console
(black) richard-26@ubuntu-laptop:~/programming/black$ cat test.py
def func(a):
    x, *y = a[0]
    return -x, *y


def main():
    a = [[1, 2, 3], [4, 5, 6]]
    print(func(a))  # >> (-1, 2, 3)


if __name__ == "__main__":
    main()
(black) richard-26@ubuntu-laptop:~/programming/black$ black test.py --diff --color
All done! ✨ 🍰 ✨
1 file would be left unchanged.
```

**Environment:**
- Python version: `3.8,1`
- _Black_ version: `version 19.10b1.dev115+g88d12f8.d20200614`
- OS version: `Ubuntu 18.04.4 LTS`

**Comment by ichard26** [comment]
I still have no idea which commit fixed this but [Black `20.08b1` has been released](https://github.com/psf/black/releases/tag/20.8b1) which along other things includes that commit.

```console
R:\Programming\black>black --version
black, version 20.8b1

R:\Programming\black>type temp.py
def func(a):
    x, *y = a[0]
    return -x, *y  # Cannot parse: 3:15:     return -x, *y


def main():
    a = [[1, 2, 3], [4, 5, 6]]
    print(func(a))  # >> (-1, 2, 3)


if __name__ == "__main__":
    main()

R:\Programming\black>black temp.py --diff --color
All done! ✨ 🍰 ✨
1 file would be left unchanged.
```
